test/e2e/hooks_test.go

```diff
@@ -344,36 +344,35 @@ spec:
   hooks:
     running:
       expression: workflow.status == "Running"
-      template: sleep
+      template: argosay-sleep-2seconds
     # This hook never triggered by following test.
     # To guarantee workflow does not wait forever for untriggered hooks.
     failed:
       expression: workflow.status == "Failed"
-      template: sleep
+      template: argosay-sleep-2seconds
   templates:
     - name: main
       steps:
       - - name: step1
-          template: exit0
+          template: argosay
 
-    - name: exit0
+    - name: argosay
       container:
-        image: alpine:latest
+        image: argoproj/argosay:v2
         command: ["/bin/sh", "-c"]
-        args: ["exit 0"]
-    - name: sleep
+        args: ["/bin/sleep 1; /argosay"]
+    - name: argosay-sleep-2seconds
       container:
-        image: alpine:latest
+        image: argoproj/argosay:v2
```

Comment:
what was the purpose of changing some of these from Alpine to `argosay`?  didn't see a mention of that in the description

Reply:
Thanks!
Not essential, just the same as the others.
https://github.com/argoproj/argo-workflows/blob/20d0923611f1df6b7147c3547aeeff6b6bfecf18/test/e2e/hooks_test.go#L89C1-L99

Reply:
Ah consistency with this rest of this _file_, gotcha.
I [searched](https://github.com/search?q=repo%3Aargoproj%2Fargo-workflows+alpine%3Alatest+language%3AGo&type=code&l=Go) the rest of the _codebase_ and found that `alpine:latest` is used with some frequency.
So didn't think that this one particularly stood out, but in the context of the file, it appears to have been inconsistent.